documentation/docs/05-load.md

```diff
@@ -264,6 +219,56 @@ Setting the same header multiple times (even in separate `load` functions) is an
 
 You cannot add a `set-cookie` header with `setHeaders` — use the [`cookies`](/docs/types#sveltejs-kit-cookies) API in a server-only `load` function instead.
 
+### Shared-only inputs
+
+In addition to the common inputs, a `load` function that runs on both the server and the client has the following inputs:
```

Comment:
I guess this helps clarify the heading, but it may be worth calling out +page and +layout for further clarity